subprojects/docs/src/docs/release/notes.md

```diff
@@ -2,6 +2,23 @@
 
 Here are the new features introduced in this Gradle release.
 
+### Dependency constraints
```

Comment:
The term "constraint" will need to be defined in the glossary as soon as https://github.com/gradle/gradle/pull/4082 is merged.

Reply:
Done.

Reply:
Thanks!